lib/common/session_model.dart

```diff
@@ -479,8 +480,13 @@ class SessionModel extends Model {
   }
 
   Plan planFromJson(Map<String, dynamic> item) {
-    final formatCurrency = NumberFormat.simpleCurrency();
-    var currency = formatCurrency.currencyName != null ? formatCurrency.currencyName!.toLowerCase() : "usd";
+    print("called plans $item");
+    final locale = Localization.locale;
+    final formatCurrency = NumberFormat.simpleCurrency(locale: locale);
+    String currency = formatCurrency.currencyName != null
+        ? formatCurrency.currencyName!.toLowerCase()
+        : "usd";
+    item.remove('expectedMonthlyPrice');
```

Comment:
Is it necessary to remove the expectedMonthlyPrice field here anymore? I tested desktop with this line removed, and it worked fine for me.

Reply:
No, I removed it.